tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,568 @@
+                    pass
+            for pvc in resources["pvcs"]:
+                try:
+                    pvc.delete()
+                except Exception:
+                    pass
+
+        request.addfinalizer(resource_teardown)
+        return resources
+
+    def verify_node_cgroup_throttling(
+        self, pod_obj, pvc_obj, qos_slice, expected_limits
```

Ruff (0.16.0)

[warning] Unused method argument: `pvc_obj`

(ARG002)